src/main/webapp/jsp/instructorFeedbackAjaxRemindParticularStudentsModal.jsp

```diff
@@ -0,0 +1,5 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8" %>
+
+<%@ taglib tagdir="/WEB-INF/tags/instructor" prefix="ti" %>
+
+<ti:ajaxRemindParticularStudentsCheckboxes />
```

Comment:
I was under the impression that constants cannot be embedded inside JSTL unless we tag-ify it and use an at tag import? I'll try to collapse it in.